server/src/assets/game/tableGenerator/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Table Generator for SWTbahn Game
+
+This application generates the configuration json files for the swtbahn game client, which are required for the signal-symbol mapping and the route suggestion mechanics.
```

Comment:
I'd prefer "python script" instead of "application"; unless there is more involved than that.